services/Doordash.mjs

```diff
@@ -51,7 +53,8 @@ class Doordash extends Service {
    * @return {Object} newly create token data
    */
   async createNewToken() {
-    const password = "988E4CDA-7367-4F00-BA57-493244C49226";
+    console.log("creating new token");
+    const password = "c27cf79d-2a03-4265-aa2c-cbe68c29cf00";
```

Comment:
is this the password we are using to log in to our acc?

Reply:
@dani5612 No this is a made up password that is used to create a guest account for doordash. I'm still experimenting with the DoorDash API since the accounts with that API do not seem to yield working Tokens. With that said, I know these are just uuid4 strings, so depending on my research, this might change to generate a new password every time.